gemrb/core/Interface.h

```diff
@@ -805,7 +810,10 @@ class GEM_EXPORT Interface
 	}
 	inline void ResetActionBar()
 	{
-		vars->SetAt("ActionLevel", 0, false);
+		auto lookup = vars.find("ActionLevel");
+		if (lookup != vars.cend()) {
+			lookup->second = 0;
+		}
```

Comment:
This can safely be unconditional. Even if the var doesn't exist at some early stage yet, we'll create it soon enough and it doesn't do any harm to have it set potentially too early.